public/app/features/alerting/unified/mocks/server/configure.ts

```diff
@@ -13,3 +16,10 @@ export const setAlertmanagerChoices = (alertmanagersChoice: AlertmanagerChoice,
   };
   server.use(alertmanagerChoiceHandler(response));
 };
+
+/**
+ * Makes the mock server respond with different folder access control settings
+ */
+export const setFolderAccessControl = (accessControl: FolderDTO['accessControl']) => {
+  server.use(folderHandler(mockFolder({ hasAcl: true, accessControl })));
+};
```

Comment:
I haven't fully decided/worked out how best to configure overriding the behaviour for _one specific_ folder's response. I won't try and solve it here as it's not strictly necessary for the tests we're currently doing.